build

```diff
@@ -304,7 +327,7 @@ function build_debian()
 	cp ./config/build/dpkg_build/postinst.sh ./output_lx/${PACK_NAME}/usr/local/bin/initRobotFrameworkAIO.sh
 
 	# Update version
-	cp ./version.txt ./output_lx/${PACK_NAME}/opt/rfwaio/
+	cp ./Readme.txt ./output_lx/${PACK_NAME}/opt/rfwaio/
```

Comment:
Hi Thong,
please let the name "version.txt".
I want to highlight that the version can be found here.
Readme can be a lot of things...
Thank you,
 Thomas